src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -437,6 +464,12 @@ def geop_tasks(geom, errback):
          nlcd_kfactor)
     ]
 
+    if geom.within(DRB):
+        definitions.append(('nlcd_streams',
+                            {'polygon': [geom.geojson],
+                             'vector': streams(geom, drb=True)},
+                            nlcd_streams_drb))
```

Comment:
That's a bit confusing, probably warrants a comment, but yes. That actually points to the JSON block that is sent to the geoprocessing service, which is defined here: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/mmw/settings/base.py#L386-L399

Both requests use the same JSON block, just different values in the `vector` field.